### ISSUE TITLE
Enable running locally

### ISSUE THREAD

**Issue by perfdude** [SamVerschueren/bragg#7]
Enable the running of Bragg code locally with node.js. Or if this is possible already, document how to do so.

**Comment by SamVerschueren** [comment]
I agree, this would be very nice. Have to check how it could be done though... Help is always welcome.

**Comment by nnn406** [comment]
The best way to accomplish that (IMO) is docker [image](https://github.com/lambci/docker-lambda)
